app/controllers/taxons_controller.rb

```diff
@@ -0,0 +1,37 @@
+class TaxonsController < ApplicationController
+  before_action :redirect_to_www, unless: :new_navigaton_enabled?
+
+  def show
+    render :show, locals: {
```

Comment:
Could we do this with an instance var? In content-tagger we do things in both ways, and the inconsistency isn't great. 

Perhaps it would be nice to create a single `NavigationPage` presenter, which exposes the methods from this hash.